src/psyclone/psyir/transformations/psy_data_trans.py

```diff
@@ -79,20 +82,24 @@ class PSyDataTrans(RegionTrans):
     '''
     # Unlike other transformations we can be fairly relaxed about the nodes
     # that a region can contain as we don't have to understand them.
-    # TODO: #415 Support different classes of PSyData calls.
     excluded_node_types = (Return,)
 
     def __init__(self, node_class=PSyDataNode):
         super(PSyDataTrans, self).__init__()
         self._node_class = node_class
 
     def __str__(self):
-        return "Insert a PSyData node."
+        return ("Create a sub-tree of the PSyIR that has a node of type "
+                "{0} at its root.").format(self._node_class.__name__)
 
     @property
     def name(self):
-        ''' Returns the name of this transformation as a string '''
-        return "PSyDataTrans"
+        ''' Returns the name of this transformation as a string.
```

Comment:
Sorry to be a pain but please could you reformat this so that it has a description plus :returns: and :rtype:.